### PR TITLE
Close contig json file if no contigs

### PR DESCRIPTION
**Pull request by rzlim08**

### CODE REVIEW

File: short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py [hunk 1]
```diff
@@ -176,6 +176,10 @@ def compute_slice(slice_idx):
                 }
             )
         )
+        # if all intermediate json files are empty, close collated json file
+        json_sizes = sum([os.path.getsize(tfn) for tfn in output_json_filenames[:-1]])
+        if json_sizes == 0:
+            ojson.write("}")
```
Comment (morsecodist) [inline short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py:182]: This seems like a typo, should it be `json` not `ojson`?

File: short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py [hunk 2]
```diff
@@ -176,6 +176,10 @@ def compute_slice(slice_idx):
                 }
             )
         )
+        # if all intermediate json files are empty, close collated json file
+        json_sizes = sum([os.path.getsize(tfn) for tfn in output_json_filenames[:-1]])
```
Comment (morsecodist) [inline short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py:180]: The `sum([` is redundant you can drop the `[`

**Review by morsecodist — CHANGES_REQUESTED** [review]

File: short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py (hunk 2 above)
Comment (rzlim08) [reply short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py]: Fixed thanks!

File: short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py (hunk 1 above)
Comment (rzlim08) [reply short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py]: sorry, forgot to open a json file. it's meant to match the opening statement.

**Pull request merged**